cmd.go

```diff
@@ -93,10 +93,6 @@ func doRemove(hfile io.Reader, out io.Writer, args []string) error {
 }
 
 func doRename(writtenFile *os.File, renameTo string) error {
-	if err := os.Chmod(writtenFile.Name(), 0644); err != nil {
```

Comment:
Do we replace this functionality somewhere, or was this incorrect?

Reply:
This happens in the `tempFile`-function now (but it copies the file mode from the existing hosts-file instead).

Reply:
Sorry, I misread that code. I guess we put it back :)